clang/test/dpct/helper_files_ref/include/dpl_extras/dpcpp_extensions.h

```diff
@@ -591,7 +592,7 @@ namespace device {
 namespace detail {
 
 template <typename... _Args> constexpr auto __joint_reduce(_Args... __args) {
-  return sycl::joint_reduce(__args...);
+  return sycl::ext::oneapi::experimental::joint_reduce(__args...);
```

Comment:
So, after this change, header files will not compile with the standard SYCL compiler. Using experimental features by default (not under the flag) doesn't look good.....

Reply:
I'll try to use this experimental features by ` --use-experimental-features=`, and add an new value for this option.
```
 --use-experimental-features=<value>     - Comma separated list of experimental features to be used in migrated code.
                                            By default, experimental features will not be used in migrated code.
                                            The values are:
    =free-function-queries                -   Experimental extension that allows getting `id`, `item`, `nd_item`, `group`, and
                                              `sub_group` instances globally.
    =local-memory-kernel-scope-allocation -   Experimental extension that allows allocation of local memory objects at the kernel
                                              functor scope
    =logical-group                        -   Experimental helper function used to logically group work-items.
    =nd_range_barrier                     -   Experimental helper function used to help cross group synchronization during migration.
```